example/app/config.yml

```diff
@@ -176,10 +176,28 @@ cumulus-from-npm:
   prefix: test-npm # used by the integration repo
   stackNameNoDash: TestNpmIntegration
 
+  iams:
```

Comment:
I think instead of copying all the iams, we can just update the prefix and stacknames here. For example:

```
cumulus-from-source:
    prefix: 'test-npm-integration'
    stackName: '{{prefix}}'